src/boilerplate.ts

```diff
@@ -281,7 +284,9 @@ export const install = async (toolbox: IgniteToolbox) => {
     ignite.log("adding boilerplate to project for generator commands")
 
     const boilerplate = parameters.options.b || parameters.options.boilerplate || "ignite-bowser"
-    await system.exec(`ignite add ${boilerplate} ${debugFlag}`)
+    const isIgniteInstalled = await system.which(`ignite`)
```

Comment:
Do you not need to compare the result?

Reply:
I could cast it with `Boolean()`. But it'll return `null` if Ignite isn't installed, which will then evaluate properly in the next line.

Reply:
Okay, I figured it would return an error string if it wasn't found.